NRLMMD-GEOIPS/geoips
# convert python script docstrings to numpy docstring format

### geoips#29 docstring google2numpy image utils - Updates the docstrings from google2numpy in image utils directory
 
# Related Issue
fixes NRLMMD-GEOIPS/geoips#29

# Reviewer Checklist

### Ensure you are logged into GitHub

### Confirm all requirements are met for pull request
* Ensure appropriate / sufficient content
    * **Title format**: geoips#<issue_num> <modified_repo_name> - \<Short description>
        * Example: geoips#8 geoips_template_plugin - Updating GEOIPS_REPO_URL to github
    * **Included Issue**: Link to related Issue included at the beginning of pull request
        * Example: NRLMMD-GEOIPS/geoips#8
    * **CHANGELOG updated**: Functionality included in pull request is ALSO referenced in CHANGELOG.md
    * **Demonstrated Testing**: Proper testing / output was demonstrated for functionality updates
    * **Included Outputs**: Imagery is included in "Output" section for new or changed product outputs (for reports!)
    * **Test Scripts/Outputs for New Functionality**: If new functionality is included in PR,
        ensure related test scripts and test outputs were included.
* Ensure all appropriate tags / attributes added along right-hand side of pull request
    * **Label**: Added appropriate descriptors
    * **Projects**: GeoIPS - All Repos and All Functionality
        * Other projects allowed as appropriate
* Ensure Related Issue is finalized appropriately (follow link above)
    
### Once all items in checklist have been confirmed:
* **Approve pull request**
    * Click "Files changed" tab
    * Click green "Review changes" button
    * Select "Approve" option
    * Add message if desired
    * Click green "Submit review" button
    * Project status will automatically be updated in Project "GeoIPS - All Repos and All Functionality" when pull request is approved.
 
# Testing Instructions
Run the following to check that all Python scripts in  `image_utils` have been updated from Google to NumPy docstring format.
```bash
pip install pydocstyle
find ./geoips/image_utils/ -type f -name "*.py" -exec pydocstyle --convention=numpy  "{}" \;
```

# Summary
* Change GeoIPS Python script documentation from Google to NumPy docstring format
* Only changes `./geoips/image_utils/`

# Output
* No test output if passes
* If fails, error codes available here [Error Codes](https://www.pydocstyle.org/en/stable/error_codes.html)

  
# Individual Commits
*
